extras/travis_build.sh

```diff
@@ -280,13 +281,8 @@ echo "MSGBLD0280: Make install completed."
     echo "+               R U N   N E S T   T E S T S U I T E                           +"
     echo "+ + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + +"
     echo "MSGBLD0290: Running make installcheck."
```

Comment:
The indentation here will conflict with the changes in [PR29](https://github.com/hakonsbm/nest-simulator/pull/29).